lib/vector-core/src/test_util.rs

```diff
@@ -0,0 +1,11 @@
+use std::fs::File;
+use std::path::Path;
+
+pub fn open_fixture(path: impl AsRef<Path>) -> crate::Result<serde_json::Value> {
+    let test_file = match File::open(path) {
+        Ok(file) => file,
+        Err(e) => return Err(e.into()),
+    };
```

Comment:
I believe this is just `let test_file = File::open(path)?;` (Yes, I realize this was probably just copied over)

Reply:
Ah, yep. It was copied over but it’s a good change to make.